components/composed/ConfirmModalBody/ConfirmModalBody.tsx

```diff
@@ -0,0 +1,38 @@
+import * as Styled from "./ConfirmModalBody.styles";
+import { ButtonControlConfirm } from "components/atomic";
+import i18next from "i18next";
+
+type ButtonControlConfirmProps = React.ComponentProps<
+  typeof ButtonControlConfirm
+>;
+
+const ConfirmModalBody = ({
+  modalBody: body,
+  handleClose,
+  onClick: onConfirm,
+  actionLabel,
+}: ConfirmModalBodyProps) => {
+  return (
+    <>
+      {body}
```

Comment:
This component should be renamed to just `ConfirmModal` but it now takes any JSX as a body prop so actions can pass in their own modal content.